recipes/sagemaker-studio-analytics-extension/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "sagemaker-studio-analytics-extension" %}
+{% set version = "0.0.19" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sagemaker-studio-analytics-extension-{{ version }}.tar.gz
+  sha256: 6b8c74f23f54d306d7d439e1cd0c47218b918b396b2a1d814600aa7dca58d28e
+
+build:
+  entry_points:
+    - sm_analytics_runtime_check=sagemaker_studio_analytics_extension.utils.runtime_check:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - boto3 >=1.26.49,<2.0
+    - sparkmagic ==0.20.4
+    - sagemaker-studio-sparkmagic-lib >=0.1.1
+    - filelock >=3.0.12
+
+test:
+  imports:
+    - sagemaker_studio_analytics_extension
+  commands:
+    - pip check || true
```

Comment:
```suggestion
    - pip check
```

Reply:
@ocefpaf 

`pip check` is currently failing for the recipe because of an inconsistency from a downstream dependency: sparkmagic.
```
2023-09-05T20:06:22.8117895Z import: 'sagemaker_studio_analytics_extension'
2023-09-05T20:06:22.8174114Z + pip check
2023-09-05T20:06:24.2076783Z autovizwidget 0.20.5 has requirement pandas<2.0.0,>=0.20.1, but you have pandas 2.1.0.
2023-09-05T20:06:24.2085694Z sparkmagic 0.20.4 has requirement nest-asyncio==1.5.5, but you have nest-asyncio 1.5.6.
2023-09-05T20:06:24.2086883Z hdijupyterutils 0.20.5 has requirement pandas<2.0.0,>=0.17.1, but you have pandas 2.1.0.
```

I checked the feedstock for sparkmagic itself which also prevents the pip check failure from breaking the build: https://github.com/conda-forge/sparkmagic-feedstock/blob/main/recipe/meta.yaml#L41

Given the issue is caused downstream, I'm happy to discuss any other recommendations you might have, but it appears any feedstock depending on sparkmagic would have this issue as well.

Reply:
It is fine to comment it out here, add a comment why, and open an issue (or better yet, a PR) downstream.

Reply:
Ack. Opened an issue and added the comment